index.html

```diff
@@ -0,0 +1,161 @@
+                <a class="button" href="contacts.html">Обратная связь</a>
+            </section>
+            <section class="appointment">
+                <h2>Записаться</h2>
+                <p class="appointment-info">Укажите желаемую дату и время и мы свяжемся с вами для подтверждения брони</p>
+                <form class="appointment-form" action="https://echo.htmlacademy.ru" method="post">
+                    <p class="appointment-item">
+                        <label for="appointment-data">Дата</label>
+                        <input id="appointment-date" type="text" name="date" value="" placeholder="08.10.2017">
+                    </p>
+                    <p class="appointment-item">
+                            <label for="appointment-time">Время</label>
+                        <input id="appointmet-time" type="text" name="time" value="" placeholder="10:00">
+                    </p>
+                    <p class="appointment-item">
+                            <label for="appointment-name">Ваше имя</label>
+                        <input id="appointment-name" type="text" name="name" value="" placeholder="Борода">
+                    </p>
+                    <p class="appointment-item">
+                            <label for="appointment-phone">Телефон</label>
+                        <input id="appointment-phone" type="tel" name="phone" value="" placeholder="+7 123 456-78-90">
+                    </p>
+                    <button class="button" type="submit">Отправить</button>
+                </form>
+            </section>
+        </main>
+        <footer class="main-footer">
+            <p class="footer-contacts">
+                Барбершоп "Бородинский" <br>
+                Адрес: г. Санкт-Петербург, ул. В. Конюшенная, д. 19/8 <br>
+                <a href="map.html">Как нас найти?</a> <br>
+                Телефон: +7 (495) 666-02-66
+            </p>
+            <div class="footer-contacts">
+                <b>Давайте дружить!</b>
+                <ul>
+                    <li><a class="social-button" href="#">Вконтакте</a></li>
+                    <li><a class="social-button" href="#">Фейскбук</a></li>
+                    <li><a class="social-button" href="#">Инстраграм</a></li>
+                </ul>
+            </div>
+            <p class="footer-contacts">
+                <b>Разработано:</b>
+                <a class="button" href="https://htlmacadamy.ru">HTML Academy</a>
+            </p>
+        </footer>
+
+        <section class="modal modal-login">
+            <h2>Личный кабинет</h2>
+            <p>Введите свой логин и пароль</p>
+            <form class="login-form" action="https://echo.htmlacademy.ru" method="post">
+                <p>
+                    <label class="visually-hidden" for="user-login">Логин</label>
+                    <input id="user-login" class="login-icon-user" type="text" name="login" placeholder="Логин">
```

Comment:
```suggestion
                    <input id="login-username" class="login-username-input" type="text" name="username" placeholder="Логин">
```

я, конечно, не вижу макет, но инпут с классом icon - это зашквар, у тебя иконкой может быть какой-нить спанчик или другой элемент.